docs/docs/mutations.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Mutations
+---
+
+```typescript
+import { httpMutation } from '@angular-architects/ngrx-toolkit';
+```
+
+```typescript
+import { rxMutation } from '@angular-architects/ngrx-toolkit';
+```
+
+```typescript
+import { withMutations } from '@angular-architects/ngrx-toolkit';
+```
+
+```typescript
+// Optional, `concatOp` is the default.
+import { concatOp, exhaustOp, mergeOp, switchOp } from '@angular-architects/ngrx-toolkit';
+```
+
+## Basic Usage
+
+The mutations feature (`withMutations`) and methods (`httpMutation` and `rxMutation`) seek to offer an appropriate equivalent to signal resources for sending data back to the backend. The methods can be used in `withMutations()` or on their own.
+
+This guide covers
+
+- Why we do not use [`withResource`](./with-resource), and the direction on mutations from the community
+- Key Features ([summary](#key-features-summary) and [in depth](#key-features-in-depth)):
+  <!-- TODO (discuss): I think it is important to know it is `HttpClient` under the hood for stuff like interceptors and global stuff -->
+  - The params to pass (via RxJS or via `HttpClient` params without RxJS)
+  - Callbacks available (`onSuccess` and `onError`)
+  - Flattening operators (`concatOp, exhaustOp, mergeOp, switchOp`)
+  - Calling the mutations (optionally as promises)
+  <!-- TODO - narrowing not working like intended? -->
```

Comment:
I made an issue for this: **https://github.com/angular-architects/ngrx-toolkit/issues/235** Leaving this in for emphasis until the round of feedback changes.